src/Core/Framework/Adapter/Translation/TranslatorCacheInvalidate.php

```diff
@@ -1,77 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Shopware\Core\Framework\Adapter\Translation;
-
-use Doctrine\DBAL\ArrayParameterType;
-use Doctrine\DBAL\Connection;
-use Shopware\Core\Framework\Adapter\Cache\CacheInvalidator;
-use Shopware\Core\Framework\DataAbstractionLayer\Event\EntityWrittenEvent;
-use Shopware\Core\Framework\Feature;
-use Shopware\Core\Framework\Log\Package;
-use Shopware\Core\System\Snippet\Aggregate\SnippetSet\SnippetSetDefinition;
-use Shopware\Core\System\Snippet\SnippetDefinition;
-use Shopware\Core\System\Snippet\SnippetEvents;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-
-/**
- * @internal
```

Comment:
the class was not directly deprecated, but was marked as internal. I removed it, as it does nothing after changing the feature flag. is that okay?